### PR TITLE
HEAL-5576 Add AWS trace header

### PR DESCRIPTION
**Pull request by jradtilbrook**
Update the log config to put the AWS trace header in the same place as other services. This will allow better filtering for related records in kibana.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v4.1.0] - 2024-11-29
```
Comment (tspencer244) [inline CHANGELOG.md:7]: Block: there is no link to this version at the bottom of the change log

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
...
+
+### Added
+
+- AWS trace ID header added to monolog record `X-Amzn-Trace-Id`
```
Comment (tspencer244) [inline CHANGELOG.md]: Block: this is not explaining exactly what this change is, e.g. it should mention that it is a top-level key in the log record

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
...
+
+### Removed
+
+- Removed `unique_id` log field
```
Comment (tspencer244) [inline CHANGELOG.md]: Block: this is not explaining exactly what the change is, .e.g. it should mention that it is a sub-key of the top-level `extra` key in the log record

File: src/Formatters/LogstashFormatter.php [hunk 4]
```diff
@@ -0,0 +1,54 @@
+<?php
```
Comment (tspencer244) [inline src/Formatters/LogstashFormatter.php:1]: Block: this class is missing test coverage

File: CHANGELOG.md (hunk 1 above)
Comment (jradtilbrook) [reply CHANGELOG.md:7]: All fixed now :pray:

**Review by tspencer244 — APPROVED** [review]

**Comment by jradtilbrook** [comment]
Thanks @tspencer244! I don't have permission to merge this, are you able to merge and create the 4.1.0 release for me?

**Pull request merged**